### DISCUSSION TITLE
A wesite that combines CI builds of both VSIX Community packages and VSIX Community extensions

### DISCUSSION THREAD

**Discussion by yannduran** [VsixCommunity/Community.VisualStudio.Toolkit#19] [Ideas]
What about if we had a website, similar to VsixGallery.com, that combined CI versions of **_VSIX Community_** extensions, and CI versions of **_VSIX Community_** packages? 

I thought about suggesting modifying VsixGellery.com to also accept Buget packages, but the name doesn't really communicate the **Community** part of **_VSIX Community_**, nor that you can find CI versions **Nuget packages** there.

It could be called something like VsixCommunityGallery.com (or something else).

There could be a **single** VS extension (like Mad's **_VSIX Gallery_** extension) that:
- installs a new _Extension Gallery_ (http://vsixcommunitytoolkit.com/extensions)
- installs a new _Nuget Package source_ (http://vsixcommunitytoolkit.com/packages)

There's currently no **one place** for CI builds of Nuget packages.

This would mean that:
-  Nuget.org wouldn't be cluttered with CI builds of the **_VSIX Community_** packages (especially as the number potentially grows)
- VsixGallery.com wouldn't be cluttered with CI builds of any **_VSIX Community_** extensions

This could be awesome! What do others think?

**Comment by yannduran** [comment]
![image](https://user-images.githubusercontent.com/2031126/115863439-c6024380-a478-11eb-93db-80c2ffde8ce7.png)
![image](https://user-images.githubusercontent.com/2031126/115863556-f6e27880-a478-11eb-8d0d-0f29c2b2c1fe.png)

**Comment by madskristensen** [comment]
There is a CI feed for the NuGet package here: https://ci.appveyor.com/nuget/community-visualstudio-toolkit

**Comment by yannduran** [comment]
Thanks for that Mads. So you don't think an updated/new/combined Community Toolkit website/extension is a good idea?

**Reply by madskristensen** [reply]
Not sure. Something that works as a landing page for the VSIX community projects is desirable, but perhaps we can use GitHub pages for that like what I've started with [the docs](https://vsixcommunity.github.io/Community.VisualStudio.Toolkit/v1/api/index.html). See the /docs folder in the repo
